documentation/content/howtocontribute.rst

```diff
@@ -157,3 +157,42 @@ If you like, you can put extra tags at the end:
 You can also create your own tag or use one that’s in common usage such as
 `Co-authored-by` or `Helped-by`.
 
+About Issues Labels
+-------------------
+We use Github issues and milestones as our tool for managing kw project. We use
```

Comment:
Nit: Maybe: "We use GitHub's issue and milestone trackers to help managing the kw project." ?

Also: s\Github\GitHub\